swift/WorkflowUI/Sources/ViewControllerDescription/DescribedViewController.swift

```diff
@@ -19,51 +19,60 @@
 import UIKit
 
 
-internal final class AnyScreenViewController: ScreenViewController<AnyScreen> {
+public final class DescribedViewController: UIViewController {
```

Comment:
This isn’t a rename, but a rewrite. I suppose they do perform mostly the same job, though.